src/libultra/os/initialize.c

```diff
@@ -1,32 +1,35 @@
 #include "global.h"
 
-u64 osClockRate = 0x0000000003B9ACA0;
-s32 osViClock = 0x02E6D354;
+#define COLD_RESET 0
```

Comment:
I'd prefer to use HARD_RESET, but it also depends if a SOFT_RESET is also used somewhere (I couldn't find it)